packages/core/src/routes/admin-user.ts

```diff
@@ -170,10 +170,10 @@ export default function adminUserRoutes<T extends AuthedRouter>(router: T) {
     koaGuard({
       params: object({ userId: string() }),
       body: object({
-        username: string().regex(usernameRegEx).optional(),
-        primaryEmail: string().regex(emailRegEx).optional(),
-        primaryPhone: string().regex(phoneRegEx).optional(),
-        name: string().nullable().optional(),
+        username: string().regex(usernameRegEx).or(literal('')).nullable().optional(),
```

Comment:
so empty string will become "null" in db?

Reply:
Yeah, I tested it on my local and it seemed so